lib/service.js

```diff
@@ -403,6 +403,31 @@ AWS.Service = inherit({
   /**
    * @api private
    */
+  getServiceClock: function getServiceClock() {
```

Comment:
I'm not sure this method name reflects what the function does. How about 'getSkewCorrectedDate' or just 'getDate'?